### ISSUE TITLE
Games with a custom extension for their ldb database file can't be launched without being put in a zip

### ISSUE THREAD

**Issue by Mimigris** [libretro/easyrpg-libretro#58]
#### Name of the game:
[Laxius Power I: Random Story](https://www.rpg-maker.fr/jeux/laxius_power.exe) (link to the installer of the game).

#### Player platform:
Windows, tested with the 0.8.0 version of the EasyRPG Player core.

#### Describe the issue in detail and how to reproduce it:
To protect themselves from potential cheaters and these kinds of persons, some games were edited to change the extensions used by the game to not allow them to be opened with RPG Maker (you could still rename each file but it's long for nothing). This is the case for the game _Laxius Power I: Random Story_ for example, where the file _RPG_RT.ldb_ was renamed to _RPG_RT.set_, meaning that it can't be opened normally in RPG Maker. EasyRPG Player is already able to handle that with its own game browser (for more information, check the section **Launching protected games** from the [blog of the version 0.7.0 of the Player](https://blog.easyrpg.org/2021/10/easyrpg-player-0-7-0-sword/)), but since the configuration in RetroArch is set to only read ldb, zip and easyrpg files, Laxius Power I: Random Story is not able to be detected, and thus, to be launched with RetroArch. One workaround that can be done if you really want to play it would be to put it in a zip file, but as said, this is just a workaround that a normal user wouldn't do.

**Comment by Ghabry** [comment]
I don't think this can be fixed in a way that is universal for various _protected_ game extensions.

Cores can only provide hardcoded file extensions to the frontend (RetroArch).

There is also no functionality in RetroArch which allows running of _folders_ (which would make sense for us and mitigate this issue), everything is just focussed on files (because RetroArch is too focussed on Roms, which are files).
